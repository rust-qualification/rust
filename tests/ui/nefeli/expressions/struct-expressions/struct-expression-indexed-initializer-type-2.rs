// Indexed initializer:  886d3a44-fcba-466d-ab0d-ab8bdc418555
struct Employee(String, i32, i32);

fn main() {
    let employee = Employee(
        "Maria".to_string(),   
        "Maria".to_string(),                       
        50000                     
    ); //~^ ERROR mismatched types
}