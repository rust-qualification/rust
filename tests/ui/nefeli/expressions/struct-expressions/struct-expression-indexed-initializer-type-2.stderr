error[E0308]: mismatched types
  --> $DIR/struct-expression-indexed-initializer-type-2.rs:7:9
   |
LL |     let employee = Employee(
   |                    -------- arguments to this struct are incorrect
LL |         "Maria".to_string(),   
LL |         "Maria".to_string(),
   |         ^^^^^^^^^^^^^^^^^^^ expected `i32`, found `String`
   |
note: tuple struct defined here
  --> $DIR/struct-expression-indexed-initializer-type-2.rs:2:8
   |
LL | struct Employee(String, i32, i32);
   |        ^^^^^^^^

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0308`.
